ballerina/target/cache/tests_cache/test_suit.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
no need this file

Reply:
Fixed with 8e43242a17ba9c0ef7c467d471137764fd2b3922